Dockerfile

```diff
@@ -3,27 +3,26 @@ FROM node:lts-alpine AS base
 
 # Assemble a pruned version of the repo containing only what's needed for router
 FROM base AS builder
-RUN apk add --no-cache libc6-compat
 RUN apk update
+RUN apk add --no-cache libc6-compat
```

Comment:
why does the order matter here exactly?

Reply:
I don't think it does tbh. Just bringing it in line with latest Dockerfie example from Turbo docs.